server/src/main/java/org/elasticsearch/index/fielddata/IndexFieldData.java

```diff
@@ -55,7 +54,7 @@
  * Thread-safe utility class that allows to get per-segment values via the
  * {@link #load(LeafReaderContext)} method.
  */
-public interface IndexFieldData<FD extends LeafFieldData> extends IndexComponent {
+public interface IndexFieldData<FD extends LeafFieldData> {
```

Comment:
Do we want to rename this, now that it's no longer an `IndexComponent`?  Or is that going to be too much churn?

Reply:
I think it'd be a bunch of churn, yeah.